mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -2291,7 +2292,7 @@ impl AtomicSwap for TakerSwap {
         if self.r().taker_fee.is_none() {
             result.push(LockedAmount {
                 coin: self.taker_coin.ticker().to_owned(),
-                amount: taker_fee_amount,
+                amount: taker_fee_amount.fee_amount(),
```

Comment:
Locked amount should be the `total_spend_amount`, this is the amount that the taker should not take into consideration when calculating balance for another order to be placed while this swap is in progress but the taker fee transaction is not sent yet (onchain balance is not yet updated).